### PR TITLE
Update to disallow attached images for fbo to have different sizes in…

### PR DESCRIPTION
**Pull request by zhenyao**
… WebGL2.

### CODE REVIEW

**Comment by zhenyao** [comment]
@RafaelCintron @jdashg @grorg Do you have any objections for this?

**Comment by RafaelCintron** [comment]
No objections here.  LGTM.

File: specs/latest/2.0/index.html [hunk 1]
```diff
@@ -3076,6 +3076,11 @@ <h2 class="no-toc">Table of contents</h2>
         This restriction ensures easy portability across OpenGL ES 3.0 supporting devices.
     </div>
 
+    <h3><a name="INCOMPLETE_DIMENSIONS">checkFramebufferStatus may return FRAMEBUFFER_INCOMPLETE_DIMENSIONS</a></h3>
```
Comment (kenrussell) [inline specs/latest/2.0/index.html]: Please also update the last modified date at the top of the spec.

**Comment by kenrussell** [comment]
LGTM. Given some of the other restrictions that have been enforced in WebGL 2.0 compared to ES 3.0, I think this is better than an inefficient implementation of this functionality.

**Comment by jdashg** [comment]
Thanks for providing rationale.

**Pull request merged**